wcfsetup/install/files/lib/bootstrap/com.woltlab.wcf.php

```diff
@@ -68,6 +72,12 @@
 
     $eventHandler->register(PackageUpdateListChanged::class, PackageUpdateListChangedLicenseListener::class);
 
+    $eventHandler->register(AcpDashboardCollecting::class, static function (AcpDashboardCollecting $event) {
+        $event->register('com.woltlab.wcf.news', new NewsAcpDashboardBox());
+        $event->register('com.woltlab.wcf.systemInfo', new SystemInfoAcpDashboardBox());
+        $event->register('com.woltlab.wcf.credits', new CreditsAcpDashboardBox());
```

Comment:
For this kind of bulk-registration it might make sense to use a fully-qualified import, like done for `RebuildWorkerCollecting`.